packages/headless/src/features/generated-answer/generated-answer-insight-analytics-actions.ts

```diff
@@ -0,0 +1,248 @@
+import {
```

Comment:
Have we defined, or are we planning to define [analytics schemas](https://github.com/coveo/analytics_schema) for these events to leverage the new protocol?

Reply:
Yes @felixpelletier that's for you :D